jackal_control/config/localization.yaml

```diff
@@ -3,27 +3,28 @@ ekf_node:
     odom_frame: odom
     base_link_frame: base_link
     world_frame: odom
-
+    publish_tf: true
     two_d_mode: true
 
     frequency: 50.0
 
     # odom0: jackal_velocity_controller/odom
     odom0: odom
-    odom0_config: [false, false, false,
-                  false, false, false,
-                  true, true, true,
+    odom0_config: [true, true, false,
+                  false, false, true,
+                  true, true, false,
                   false, false, true,
                   false, false, false]
     odom0_differential: false
     odom0_queue_size: 10
 
     imu0: imu/data
     imu0_config: [false, false, false,
-                  true, true, true,
                   false, false, false,
-                  true, true, true,
-                  false, false, false]
-    imu0_differential: true
+                  true, false, false,
```

Comment:
How come there is one true here? Isn't it X velocity which isn't measured by the IMU?